Use cases (#2438)
ethereum/ethereum-org-website: src/content/nft/index.md

---
title: Non-fungible tokens (NFT)
description: An overview of NFTs on Ethereum
lang: en
template: use-cases
sidebar: true
sidebarDepth: 2
image: ../../assets/eth2/core.png
summaryPoints:
  [

>     "A way to turn anything unique into an Ethereum-based asset.",

Reviewer: Maybe "A way to represent anything unique as an Ethereum-based asset." would be more clear.

    "NFTs are giving more power to content creators than ever before.",
    "Powered by smart contracts on the Ethereum blockchain.",
  ]
---

> NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans.

Reviewer: It'd be nice to add some hard data to this, e.g. according to this site "Total Crypto Art Value" is now over $100M: https://cryptoart.io/data

Reviewer: I feel like Josh's article does a good job of putting numbers behind some examples: https://jjmstark.medium.com/the-year-in-ethereum-2020-98123e5f160d

Author: I've added a link to that cryptoart site in the further reading section. I don't want to add it to the intro for a couple reasons:
- we're not just talking about art
- any figures we pull out will have to be updated regularly

Here's what you need to know about NFTs.

> ## What's an NFT?

Reviewer: Worth [defining/explaining "fungibility"](https://www.investopedia.com/terms/f/fungibility.asp) given that it's part of the name? Or was the thinking to intentionally avoid it, given it's a relatively esoteric term to begin with?

> NFTs are tokens that we can use to represent a record of ownership. They let us tokenise art, collectibles, even purely physical items like designer trainers/sneakers. They can only have one owner at a time and they're protected by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.

Reviewer: > They let us tokenise art, collectibles, even purely physical items like designer trainers/sneakers.

This is solid but IMO potentially paints too narrow of a picture - that NFTs are primarily focused around art/design/games. Perhaps we throw in a mention of real estate (title to a house?) or financial securities (a government bond?) to demonstrate how broad these applications could be?

_If Andy Warhol had been born in the late 90s, he probably would have minted Campbell's Soup as an NFT. It's only a matter of time before Kanye puts a run of Yeezys on Ethereum. And one day owning your car and your rental agreement might be managed with NFTs._

### Examples

> THe NFT world is relatively new. In theory, the scope for NFTs is anything that is unique that needs provable ownership. Here are some examples of NFTs that exist today, to help you get the idea:

Reviewer: ```suggestion
The NFT world is relatively new. In theory, the scope for NFTs is anything that is unique that needs provable ownership. Here are some examples of NFTs that exist today, to help you get the idea:
```

> - A unique digital artwork.

Reviewer: Should we link out to examples here? Or is that a slippery slope?

Author: No, I think examples would work. Maybe the first of a varieties...

Author: DECISION: Add links

- #23/#100 of a limited edition run of designer sneakers.
- An in-game item.
- An essay.
- A digital collectible like a trading card.

## How do NFTs work?

NFTs have some special properties:

- each token is unique.

> - they're not interchangeable with another NFT in the same run.

Reviewer: Not sure what you mean by "in the same run" here.

- each token has an owner and this information is easily verifiable.

> - you can't own a % of an NFT token in the same way you can own 0.0001 ETH.

Reviewer: Perhaps "indivisible" is a simple way to explain this?

Reviewer 2: Yeah, I like the point but the `%` threw me off a little

Author: apparently there are projects out there that let you own a percentage of an NFT so I'll remove this point...

## What's an NFT?

NFTs are tokens that we can use to represent a record of ownership. They let us tokenise unique items like art, collectibles, even real estate. They can only have one owner at a time and they're protected by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.

NFT stands for non-fungible token. Non-fungible is an economic term that you could use to describe things like your furniture, a song file, or your computer. These things are not exchangeable for other items because they have unique properties.

Fungibile items on the other hand can be exchanged as their value defines them rather than their unique properties. For example ETH is fungible because 1 ETH is exchangeable for another ETH.

### NFT examples

THe NFT world is relatively new. In theory, the scope for NFTs is anything that is unique that needs provable ownership. Here are some examples of NFTs that exist today, to help you get the idea:

- A unique digital artwork.
- #23/#100 of a limited edition run of designer sneakers.
- An in-game item.
- An essay.
- A digital collectible like a trading card.
- A domain name...

> 

Reviewer 2: Could also represent right to access something like an event, like a digital ticket

Author: great idea. Can you think of any projects that have done this?

Reviewer 2: Not a lot of live events lately, but I have seen interviews with Mark Cuban who owns the Dallas Mavericks, and has commented on how he wants to use NFT's for tickets to get royalties with any re-sale. Don't think they've done it yet though. 

Another semi-example would be POAPs being used to gain special access or reduced prices at DevCon.

In a lot of regular games you can purchase items for you to use in your game. But if that item was an NFT you could recoup your money by selling it on when you're done with the game. You might even make a profit if that item becomes more desirable.

For game developers – as issuers of the NFT – they could earn a royalty every time an item is re-sold in the open marketplace. This creates a more mutually-beneficial business model where both players and developers earn from the secondary NFT market.

This also means that if a game is no longer maintained by the developers, the items you've collected remain yours under your control.

Ultimately the items you grind for in-game can outlive the games themselves. Even if a game is no longer maintained, your items will always be under your control. This means in-game items become digital memorabilia and have a value outside of the game.

Decentraland, a virtual reality game, lets you buy NFTs representing virtual parcels of land that you can use as you see fit.

<InfoBanner emoji=":eyes:">
  <div>Check out Ethereum games, powered by NFTs...</div>
  <ButtonLink to="/dapps?category=gaming">Explore NFT games</ButtonLink>
</InfoBanner>

> ### Physical items

Reviewer 2: Perhaps a comment on the future of titles/deeds to homes or cars going on the blockchain. You could buy a car/home with ETH and retrieve the deed via NFT in the same transaction in an atomic swap, and theoretically your Ethereum wallet could become the "key" to your new car or smart home (haven't seen real life examples of this yet, but seems very possible).

Reviewer 2: Physical items of value are also being minted as NFTs to be used as collateral in defi... see below

...

### Making ethereum addresses more memorable

One of the most useful NFTs developed so far is the Ethereum Name Service. This uses an NFT to provide your Ethereum address with an easier-to-remember name like `mywallet.eth`. This means you could ask someone to send you ETH via `mywallet.eth` rather than `0x123456789.....`.

This works in a similar way to a website domain name which makes an IP address more memorable. And like domains, ENS names have value, usually based on length and relevance. With ENS you don't need a domain registry in order to facilitate the tranfer of ownership. Instead, ENS names can be traded on NFT marketplaces with ease.

ENS names can also point to websites – not just wallets. For example, if you visit ethereum.eth in a web3 browser, it will open up a version of this website.

## NFTs and Defi

NFT-backed loans.

> Minting insurance cover as an NFT that's tradeable on secondary markets....

Reviewer 2: Perhaps something about using NFT's as collateral to obtain loans. If you don't have much cash or crypto, but you own valuables, these can potentially be used as collateral to obtain a loan in defi.

- Ethereum never goes down, meaning your tokens will always be available to sell.

<!-- ## How to NFT

### Buy NFTs

- Foundation
- OpenSea
- Rarible

Want to show off your collection, we recommend Rainbow wallet. They even support rich media NFTs.

<>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">🎶 hear your NFTs 🌈🔜✨🤫𝙥𝙡𝙪𝙨 3⃞🅳 <a href="https://t.co/IasBswM2fT">pic.twitter.com/IasBswM2fT</a></p>&mdash; Rainbow (@rainbowdotme) <a href="https://twitter.com/rainbowdotme/status/1361161839099445251?ref_src=twsrc%5Etfw">February 15, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</>

### Mint NFTs

Want to turn your content into a NFT?

- Foundation
- Other platforms... -->

### Build with NFTs

NFTs are built using a consistent standard known as [ERC-721](/developers/docs/standards/tokens/erc-721/). Read more about it in our docs.

## NFT apps

> [Add links to NFT apps categorised in the same way as the dapps page]

Reviewer 2: @ryancreatescopy Did you want to link directly to projects? Or linking to each "category" on the `/dapps` page?

Author: directly to projects. One less step if users want to go straight there rather than explore the dapps from the dapps page

## How do NFTs work?

NFTs have some special properties:

- each token is unique.
- they're not interchangeable with another NFT. For example 1 ETH is exactly the same as another ETH. This isn't the case with NFTs.
- each token has an owner and this information is easily verifiable.
- they live on Ethereum, so are compatible with every single NFT market on Ethereum.

In other words, if you own an NFT:

- only you own it.
- you can easily prove you own it.
- no one can manipulate it in any way.
- you can sell it on, and in some cases this will earn the original creator a % of resale royalties.

And if you create an NFT:

- you can easily prove you're the creator.
- you determine the scarcity.
- you can earn royalties every time it's sold.
- you can sell it on any NFT market or peer-to-peer. You're not locked in to any platform and you don't need anyone to intermediate.

### Fractional ownership

Of course not every single item has a single owner. Songs can have multiple writers, you might own your property with your partner, and you might want to sell a percentage of ownership in your film project to help get it off the ground.

For this reason, NFTs also support multiple owners.

> ## What are NFTs used for?

Author: There are differences between ERC720 and ERC1155 – do you think this difference is worth explaining on this page? Or should we just reference it in the build section at the end? Thoughts @samajammin @wackerow ?

---
title: Non-fungible tokens (NFT)
description: An overview of NFTs on Ethereum
lang: en
template: use-cases
emoji: ":frame_with_picture:"
sidebar: true
sidebarDepth: 2
image: ../../assets/infrastructure_transparent.png
summaryPoints:
  [
    "A way to represent anything unique as an Ethereum-based asset.",
    "NFTs are giving more power to content creators than ever before.",
    "Powered by smart contracts on the Ethereum blockchain.",
  ]
---

NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans. But digital art is only one way to use NFTs. Really they can be used to represent ownership of any unique asset, like a deed for an item in the digital or physical realm.

_If Andy Warhol had been born in the late 90s, he probably would have minted Campbell's Soup as an NFT. It's only a matter of time before Kanye puts a run of Yeezys on Ethereum. And one day owning your car might be managed with NFTs._

> <InfoBanner isWarning>NOTE: we don't currently explain the problems NFTs solve. On defi we talk about the problems with the tradfi system and on daos we talk about tthe difficulties of trusting strangers. The internet today isn't set up for ownership, but how can we easily explain this?</InfoBanner>

Reviewer: What problems are NFTs solving? Here's a few that stick out to me:

1. Digital scarcity
    * No provable uniqueness in the current internet… music/video copy is the same file as the original
2. Digital ownership
    * Hard to verify ownership of something. Typically requires some "credible" central party assuring who owns what.
3. Interoperability
    * In current internet world, there is none. e.g. all concert ticket companies… Eventbrite, Ticketmaster… none of them are interoperable with our wallets. Need to build their own apps & marketplaces & partnerships with venues 
    * NFTs can "plug in" to existing infrastructure (e.g. DEXs to trade, DeFi protocols to borrow money using NFTs as collateral). Empowers creators - no longer reliant on services for this infrastructure / distribution & restrained by their terms of use / geographic limitations

Reviewer: Along w/ #3, because there's interoperability, there's global liquidity when you want to exit 😄

## What's an NFT? {#what-are-nfts}

> NFTs are tokens that we can use to represent a record of ownership. They let us tokenise unique items like art, collectibles, even real estate. They can only have one owner at a time and they're protected by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.

Reviewer: ```suggestion
NFTs are tokens that we can use to represent a record of ownership. They let us tokenise unique items like art, collectibles, even real estate. They can only have one owner at a time and they're secured by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.
```

Author: I guess with fractional ownership "They can only have one owner at a time" is a bit misleading.

NFT stands for non-fungible token. Non-fungible is an economic term that you could use to describe things like your furniture, a song file, or your computer. These things are not exchangeable for other items because they have unique properties.

> Fungibile items on the other hand can be exchanged as their value defines them rather than their unique properties. For example ETH is fungible because 1 ETH is exchangeable for another ETH.

Reviewer: My 2 gwei: I think it would probably help to first define fungible items, e.g. cash, oil, a loaf of bread.

- A unique digital artwork.
- A unique shirt in a limited-run fashion line.
- An in-game item.
- [An essay](https://zora.co/0x517bab7661C315C63C6465EEd1b4248e6f7FE183/145).
- [A digital collectible](https://www.larvalabs.com/cryptopunks/details/1).
- [A domain name](https://app.ens.domains/name/ethereum.eth)
- A ticket that gives you access to an event or a coupon – these items are often known as semi-fungible. They are fungible until they are redeemed, at which point they can't be traded for a non-redeemed version.

### ethereum.org examples {#ethereum-org-examples}

We've used NFTs to give back to our contributors and store our website outside of centralized servers.

#### POAPs (Proof of attendance protocol) {#poaps}

If you contribute to ethereum.org, you can claim a POAP NFT. These are collectibles that prove you participated in an event. Some crypto meetups have used POAPs as a form of ticket to their events. [More on contributing](/contributing/#poap).

![ethereum.org POAP](../../assets/use-cases/poap.png)

#### ethereum.eth {#ethereum-dot-eth}

> This website is accessible via the ethereum.eth ENS (Ethereum name service) name. With the content hosted on IPFS, a decentralized storage solution, this website is actually an NFT.

Reviewer: + more cool things you can do with an ENS name:

> You can use your ENS name to receive any cryptocurrency, token, or blockchain-based item (like NFTs), point to a decentralized website (how to surf the decentralized web, e.g. almonit.eth.link), or store profile information like an avatar, email address, or Twitter handle. Since it can store any arbitrary information, ultimately any person or project can have their own ENS record.

https://medium.com/the-ethereum-name-service/reminder-that-ens-names-are-nfts-what-this-means-for-your-names-b7bcbea8715e#:~:text=ETH%20names%20on%20ENS%20are,So%20are%20DNS%20.&text=This%20means%20they%20work%20in,In%20fact%2C%20.

Ethereum makes it possible for NFTs to work for a number of reasons:

- Transaction history and token metadata is publicly verifiable – it's simple to prove ownership history.
- Once a transaction is confirmed, it's nearly impossible to manipulate that data to "steal" ownership.
- Trading NFTs can happen peer-to-peer without needing platforms that can take large cuts as compensation.
- All Ethereum products share the same "backend". Put another way, all Ethereum products can easily understand each other – this makes NFTs portable across products. You can buy an NFT on one product and sell it on another easily. As a creator you can list your NFTs on multiple products at the same time – every product will have the most up-to-date ownership information.
- Ethereum never goes down, meaning your tokens will always be available to sell.

## The environmental impact of NFTs {#environmental-impact-nfts}

NFTs are growing in popularity which means they're also coming under increased scrutiny – especially over their carbon footprint.

To clarify a few things:

- By pressing the button to create or buy an NFT you are not directly releasing carbon into the atmosphere.

> - NFTs aren't directly increasing the carbon footprint of Ethereum.

Reviewer: I feel like these two bullets are saying the same thing. I think this one is more to the point.

Here's where things are going to get a little technical, but bear with us...

### Don't blame it on the NFTs

> The whole NFT ecosystem works because Ethereum is decentralized and tamper-proof.

Reviewer: I'd personally go with "secure" vs "tamper-proof"

> Decentralized meaning you and everyone else can verify you own something without trusting or granting custody to a centralized third party who can impose their own rules at will. It also means your NFT is portable across many different products and markets.

Reviewer: ```suggestion
Decentralized meaning you and everyone else can verify you own something without trusting or granting custody to a third party who can impose their own rules at will. It also means your NFT is portable across many different products and markets.
```

Tamper-proof meaning no one can copy/paste your NFT or steal it.

> These qualities of Ethereum makes digitally owning unique items and monetising your content fairly possible.

Reviewer: ```suggestion
These qualities of Ethereum makes digitally owning unique items and monetising your content fairly possible but comes at a cost. Blockchains like Bitcoin and Ethereum are energy intensive right now because it takes a lot of energy to preserve these qualities. If it was easy to rewrite Ethereum's history to steal NFTs or cryptocurrency, the system collapses.
```
I'd merge these paragraphs.

Blockchains like Bitcoin and Ethereum are energy intensive right now because it takes a lot of energy to preserve these qualities. If it was easy to rewrite Ethereum's history to steal NFTs or cryptocurrency, the system collapses.

<!-- [Need: how do centralized systems do this and why is it worse?]
 -->

> #### The work in minting an NFT

Reviewer: Overall I think this section is great.

When you mint an NFT, a few things have to happen:

- it needs to be confirmed as an asset on the blockchain.
- the owner's account balance must be updated to include that asset. This makes it possible for it to then be traded or verifiably "owned".

> - the transactions that confirm the above need to be added to a block and "immortalised" on the chain.

Reviewer: These first 3 points seem to all say the same thing, though perhaps that helps to clarify it 🤷

Author: they're not sequential. I believe it's necessary to paint the entire picture of what's happening.

- the block needs to be confirmed by everyone in the network as "correct". This consensus is what makes it possible for NFT creators to maximise their earnings – there are no intermediaries needed.

These tasks are done by miners. And they let the rest of the network know about your NFT and who owns it. This means mining needs to be sufficiently difficult, otherwise anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership. There are lots of incentives in place to make sure miners are acting honestly.

[More on mining](/developers/docs/consensus-mechanisms/pow/)

#### Making the NFT tamper-proof

Mining difficulty comes from the fact that it takes a lot of computing power to create new blocks in the chain. Importantly, blocks are created consistently, not just when they're needed. They're created every 12 seconds or so.

> This is important for making Ethereum tamper-proof, one of the qualities that makes NFTs possible. The more blocks the more secure the chain. If your NFT was created in block #600 and a hacker were to try and change themselves to the creator of your NFT, it would immediately change the digital fingerprint of all subsequent blocks, #601, #602, #603, and so on. That means anyone running Ethereum software would immediately be able to tell and prevent it from happening.

Reviewer: ```suggestion
This is important for making Ethereum tamper-proof, one of the qualities that makes NFTs possible. The more blocks the more secure the chain. If your NFT was created in block #600 and a hacker were to try and change themselves to the creator of your NFT, it would immediately change the digital fingerprint of all subsequent blocks, #601, #602, #603, and so on. That means anyone running Ethereum software would immediately be able to detect and prevent it from happening.
```

However this means that computing power needs to be used constantly. It also means that a block that contains 0 NFT transactions will still have roughly the same carbon footprint, because computing power will still be consumed to create it. Other non-NFT transactions will fill the blocks.

#### Blockchains are energy intensive, right now

So yes, there is a carbon footprint associated with creating blocks by mining – and this is a problem for chains like Bitcoin too – but it's not directly the fault of NFTs.

A lot of mining is done using renewable energy sources or untapped energy in remote locations. And there is the argument that the industries that NFTs and cryptocurrencies are disrupting have huge carbon footprints too. But just because existing industries are bad, doesn't mean we shouldn't strive to be better.

> And we are. Ethereum is going through some upgrades to make using Ethereum (and by virtue, NFTs) more energy efficient. And that's always been the plan.

Reviewer: ```suggestion
And we are. Ethereum is evolving to make using Ethereum (and by virtue, NFTs) more energy efficient. And that's always been the plan.
```

### A greener future...

> For as long as Ethereum has been around, the energy-consumption of mining has been a huge focus area for developers and researchers. And the vision has always been to replace it as soon as possible.

Reviewer 2: The rewards from mining offers financial incentive to seek out and invest in cheaper, more efficient energy solutions, that aren't necessarily tethered to existing dense civilization. Mining rigs can be built and connected to the network nearly anywhere in the world. This investment in renewable energy infrastructure can be developed into the next generation of greener civilizations. 

(another general proof-of-work argument... more applies to Bitcoin long term, but thought this may have a place in this "greener future" section)

[More on Ethereum's vision](/eth2/vision/)

This vision is being delivered right now.

#### The end goal: Eth2

Ethereum is currently going through a series of upgrades, known as Eth2, that will replace mining with [staking](/staking/). This will remove computing power as a security mechanism, and reduce Ethereum's carbon footprint by several orders of magnitude. Stakers commit funds instead of computing power to secure the network.

> The energy-cost of Ethereum will become the cost of running a home computer multiplied by the number of nodes in the network. If there are 10,000 nodes in the network and the cost of running a home computer is roughly 400kWh per year. That's 4gWh per year for the entire network. By comparison, [mining is estimated to consume ~24tWh](https://digiconomist.net/ethereum-energy-consumption).

Reviewer: I love stats. I think it'd be most powerful to explain this difference as a percentage fraction.

> [@paul should we compare this to visa? This website estimates that 100,000 visa transactions costs 149 gwh: https://www.statista.com/statistics/881541/bitcoin-energy-consumption-transaction-comparison-visa/ – if we figure out how many ttransactions in a year that visa does, we can compare it to the total energy in PoS?]

Reviewer: > if we figure out how many transactions in a year that visa does

Or calculate how many much a transaction costs on each.

Mining: 24000000000 kwH
Staking: 4000000 kwH

> 4000000 / 24000000000 \* 100 = 0.01666666667% 1/10,000th of a percent of the energy consumption.

Reviewer: Ah there it is!

> Importantly this improves the energy efficiency while preserving the decentralization and security. Many other blockchains out there might already use some form of staking, but they might be secured by a select few stakers, not the 1000s that Ethereum will have. The more decentralization, the more secure the system.

Reviewer: ```suggestion
Importantly this improves the energy efficiency while preserving the decentralization and security. Many other blockchains out there might already use some form of staking, but they might be secured by a select few stakers, not the thousands that Ethereum will have. The more decentralization, the more secure the system.
```

[More on Eth2](/eth2/)

> #### Rollups in the meantime...

Reviewer: I think this & the EIP section, while informative, could be cut.
